packages/store/src/types.js

```diff
@@ -741,12 +749,12 @@
 
 /** @typedef {(...args: any[]) => any} Method */
 
-// TODO parameterize this to match the behavior object it guards
 /**
+ * @template {Record<any, MethodGuard>} M
```

Comment:
```suggestion
 * @template {Record<any, MethodGuard>} [M=Record<any, MethodGuard>]
```
When a type param's default is the same as its supertype constraint, is there a less redundant way to say this?

Reply:
that's the most concise I know of. added.